boost-maven/boost-boms/boost-mp21-apis-bom/pom.xml

```diff
@@ -0,0 +1,81 @@
+<!-- Copyright (c) 2018 IBM Corporation and others. All rights reserved. 
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+    and is available at http://www.eclipse.org/legal/epl-v10.html Contributors: 
+    IBM Corporation - initial API and implementation -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>boost.boosters</groupId>
+    <artifactId>boost-mp21-apis-bom</artifactId>
+    <!-- Versioned separately from the main project -->
+    <version>0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <description>Booster MP21 APIs Bom</description>
+
+    <properties>
+        <!-- MicroProfile 2.1 specs  -->
+        <config-version>1.3</config-version>
+        <ft-version>1.1</ft-version>
+        <health-version>1.0</health-version>
+        <metrics-version>1.1</metrics-version>
+        <jwt-version>1.1</jwt-version>
+        <openapi-version>1.0</openapi-version>
+        <rest-client-version>1.1</rest-client-version>
+        <opentracing-version>1.2</opentracing-version>
+
+    </properties>
+
+	<dependencyManagement>
+		<dependencies>
+            <dependency>
+                <groupId>boost.boosters</groupId>
```

Comment:
See previous comment about reusing Eclipse BOMs... but if we were to use our EE API BOMs, we probably need to use an **import** here (unless I'm missing the purpose).